internal/config/config.go

```diff
@@ -28,11 +28,6 @@ import (
 	xoauth2 "golang.org/x/oauth2"
 )
 
-var (
-	UuidRE         = regexp.MustCompile("^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-4[a-fA-F0-9]{3}-[8|9|aA|bB][a-fA-F0-9]{3}-[a-fA-F0-9]{12}$")
```

Comment:
This, too, looks like it's not technically required in order to move the Metal SSH key code.